src/app/organizations/manage/policy-edit.component.ts

```diff
@@ -186,6 +186,18 @@ export class PolicyEditComponent implements OnInit {
                 }
                 return true;
 
+            case PolicyType.SingleOrg:
+                if (this.enabled) { // Don't need prevalidation checks if submitting to enable
```

Comment:
Let's move the code comment up above the `if (...)` statement vs. next to it.

Reply:
Moved.